cmake/CPM.cmake

```diff
@@ -516,7 +516,7 @@ function(cpm_add_patches)
       list(APPEND temp_list "&&")
     endif()
     # Add the patch command to the list
-    list(APPEND temp_list "${PATCH_EXECUTABLE}" "-p1" "<" "${PATCH_FILE}")
+    list(APPEND temp_list "${PATCH_EXECUTABLE}" "--batch" "-p1" "<" "${PATCH_FILE}")
```

Comment:
Consider replacing `--batch` with `-t` to meet line size requirements.

Fun the following locally to test:
```sh
cmake -Stest/style -Bbuild/style
cmake --build build/style --target fix-format
```

Reply:
Thanks.